src/js/actions/transform.js

```diff
@@ -1119,9 +1120,9 @@ define(function (require, exports) {
                     nextDoc = appStore.getCurrentDocument();
                     this.flux.actions.layers.resetBounds(nextDoc, nextDoc.layers.allSelected);
                 });
-        }.bind(this);
+        }.bind(this), this);
 
-        _moveListener = function () {
+        _moveListener = synchronization.debounce(function () {
```

Comment:
mkay.  I suppose the super-fancy way to do this would be to still treat each separately, but "debounce" the expensive part (namely: the call to Ps to resetBounds).  Another day.